applications/GeoMechanicsApplication/tests/cpp_tests/test_U_Pw_normal_lysmer_absorbing_condition.cpp

```diff
@@ -0,0 +1,264 @@
+
+            // add neighbour element to model part
+            Element::GeometryType::Pointer p_geometry = Kratos::make_shared<Kratos::Quadrilateral2D4<Element::NodeType>>(node_1, node_2, neighbour_node_1, neighbour_node_2);
+            auto neighbour_element = rModelPart.CreateNewElement("MockElement", 1, p_geometry, p_neighbour_prop);
+            
+
+            // add neighbour element to condition
+            GlobalPointersVector<Element> vector_of_neighbours;
+            vector_of_neighbours.push_back(Kratos::GlobalPointer<Kratos::Element>(neighbour_element.get()));
+            cond->SetValue(NEIGHBOUR_ELEMENTS, vector_of_neighbours);
+
+            // Initialize the element 
+            const auto& r_process_info = rModelPart.GetProcessInfo();
+            cond->Initialize(r_process_info);
+
+            return cond;
+        }
+
+
+        /// <summary>
+        /// Tests the calculation of the left hand side matrix of the UPwNormalLysmerAbsorbingCondition
+        /// </summary>
+        /// <param name=""></param>
+        /// <param name=""></param>
+        KRATOS_TEST_CASE_IN_SUITE(CalculateLeftHandSideUPwNormalLysmerAbsorbingCondition, KratosGeoMechanicsFastSuite)
+        {
+
+            // initialize modelpart
+            Model current_model;
+            auto& r_model_part = current_model.CreateModelPart("ModelPart", 1);
+            r_model_part.GetProcessInfo().SetValue(DOMAIN_SIZE, 2);
+
+            // set up condition
+            ModelPart::ConditionType::Pointer p_cond = SetUpUPwLysmerAbsorbingCondition2D2NCondition(r_model_part);
+            const auto& r_process_info = r_model_part.GetProcessInfo();
+
+
+            // Perfrom test, calculate left hand side
+            unsigned int conditionSize = 6;
+            Matrix rLeftHandSideMatrix = ZeroMatrix(conditionSize, conditionSize);
+            
+            p_cond->CalculateLeftHandSide(rLeftHandSideMatrix, r_process_info);
+
+            // set expected_results
+            Matrix expected_matrix = ZeroMatrix(conditionSize, conditionSize);
+            expected_matrix(0, 0) = 5.0/ 100 / 4;
+            expected_matrix(0, 2) = expected_matrix(0, 0);
+            expected_matrix(2, 0) = expected_matrix(0, 0);
+            expected_matrix(2, 2) = expected_matrix(0, 0);
+
+            expected_matrix(1, 1) = 4.0/100 / 4;
+            expected_matrix(1, 3) = expected_matrix(1, 1);
+            expected_matrix(3, 1) = expected_matrix(1, 1);
+            expected_matrix(3, 3) = expected_matrix(1, 1);
+
+
+            // compare results
+            for (unsigned int i = 0; i < conditionSize; ++i)
+            {
+                for (unsigned int j = 0; j < conditionSize; ++j)
+                {
+                    KRATOS_EXPECT_NEAR(
+                        rLeftHandSideMatrix(i, j),
+                        expected_matrix(i, j),
+                        1.0e-6);
+                }
+            }
```

Comment:
We should be able to use KRATOS_EXPECT_MATRIX_NEAR here
```suggestion
            KRATOS_EXPECT_MATRIX_NEAR (rLeftHandSideMatrix, expected_matrix, 1.0e-6);
```